expirable_store.go

```diff
@@ -57,12 +68,18 @@ func newExpirableStore(maxSize int, maxEntryTTL time.Duration, o ...Option) (*ex
 		return nil, fmt.Errorf("%s: number of buckets must be greater than zero: %w", op, ErrInvalidNumberBuckets)
 	}
 
-	bucketTTL := (maxEntryTTL) / time.Duration(opts.withNumberBuckets-1)
+	var bucketTTL time.Duration
+	switch opts.withNumberBuckets {
+	case 1:
+		bucketTTL = maxEntryTTL
+	default:
+		bucketTTL = (maxEntryTTL) / time.Duration(opts.withNumberBuckets-1)
```

Comment:
```suggestion
		bucketTTL = maxEntryTTL / time.Duration(opts.withNumberBuckets-1)
```
Any reason for the `( )` around `maxEntryTTL`?